pkg/ingester/owned_series_test.go

```diff
@@ -1447,7 +1447,9 @@ func TestOwnedSeriesIngesterRingStrategyRingChanged(t *testing.T) {
 
 	t.Run("change of state is not interesting", func(t *testing.T) {
 		updateRingAndWaitForWatcherToReadUpdate(t, wkv, func(desc *ring.Desc) {
-			desc.AddIngester(instanceID2, "localhost:22222", "zone", []uint32{4, 5, 6}, ring.LEAVING, time.Now())
+			ingester2 := desc.Ingesters[instanceID2]
```

Comment:
Can you add a comment explaining why we're changing the state like this instead of using `AddIngester()`

Reply:
sure thing: https://github.com/grafana/mimir/pull/8050